tools/js-sdk-release-tools/src/llc/generateReadmeMd.ts

```diff
@@ -0,0 +1,146 @@
+import * as fs from "fs";
+import * as path from "path";
+import {logger} from "../utils/logger";
+import {getClientName, getServiceNameFromPackagePath, getServicePrintName} from "./utils";
+
+export function generateReadmeMd(packagePath: string, packageName: string) {
+    if (fs.existsSync(path.join(packagePath, 'README.md'))) {
+        logger.logGreen(`README.md already exists, and we don't generate it.`)
+        return;
+    }
+
+    const serviceName = getServiceNameFromPackagePath(packagePath);
+    const clientName = getClientName(packagePath);
+
+    const content = `# Azure ${getServicePrintName(packageName)} REST client library for JavaScript
+
+[Service Description]
```

Comment:
Instead of hard-coding the string here I'd highly recommend you take a file path from the js repo with a MD file that contains the content plus some markers to replace with command information. It will be a lot easier to maintain that file in the future.